assets/js/countdown.js

```diff
@@ -67,8 +71,8 @@ function initializeClock(id, startTime, endTime) {
 } // end of setting up clock
 
 window.onload = function() {
-	let hackingBegins = "February 24 2018 12:00:00 GMT-0700";
-	let hackingEnds = "February 25 2018 11:59:59 GMT-0700";
+	let hackingBegins = "{{ site.start_time }}";
```

Comment:
Any chance to use the filter `date` as seen here: https://github.com/HackCU/Startups2Students/blob/master/index.html#L95

This would avoid any bad formatting in the config file.